src/sync/sync_manager.cpp

```diff
@@ -286,25 +287,21 @@ bool SyncManager::perform_metadata_update(std::function<void(const SyncMetadataM
 }
 
 std::shared_ptr<SyncUser> SyncManager::get_user(const std::string& identity,
```

Comment:
I changed this method signature to use my new identifier struct type, so hopefully this won't be an issue anymore.